src/test/java/nextstep/member/domain/GithubClientTest.java

```diff
@@ -0,0 +1,47 @@
+package nextstep.member.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import nextstep.member.domain.dto.GithubProfileResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@DisplayName("깃허브 클라이언트 테스트")
+class GithubClientTest {
+
+    private GithubClient githubClient;
+
+    @BeforeEach
+    void init() {
+        githubClient = new GithubClient("github-client-id", "github-client-secret",
+                "http://localhost:8080/github/login/oauth/access_token", "http://localhost:8080/github/user");
+    }
```

Comment:
직접 생성하는 대신에 Spring 컨테이너로부터 주입 받아서 사용해도 될 것 같습니다.
```suggestion
    @Autowired
    private GithubClient githubClient;
```